hooks/useTagsList.js

```diff
@@ -1,5 +1,45 @@
 import { useState, useEffect } from 'react';
-import { getTagsListForCategory } from '../utilities/dotcms';
+
+const getTagsListForCategory = async (category) => {
+    const data = {
+        query: {
+            query_string: {
+                query: `+contentType:product +categories:${category}`
+            }
+        },
+        aggs: {
+            tag: {
+                terms: {
+                    field: 'tags',
+                    size: 100
+                }
+            }
+        },
+        size: 0
+    };
+
+    const options = {
+        method: 'post',
+        body: JSON.stringify(data),
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    };
+
+    const results = await fetch(`${process.env.NEXT_PUBLIC_DOTCMS_HOST}/api/es/search`, options);
```

Comment:
`${process.env.NEXT_PUBLIC_DOTCMS_HOST}` I think we don't need this because we add this to the rewrite in next.config: https://github.com/dotCMS/dotcms-spa/pull/87/files#diff-5d0c276360a637d1b787a57760665fbeR16